test/uncaught-exception-from-handler.test.ts

```diff
@@ -49,7 +49,7 @@ test('uncaught exception in immediate after task yields error event', async () =
   pool.threads[0]!.ref?.()
 
   // This is the main aassertion here.
-  expect((await errorEvent)[0]!.message).toEqual('not_caught')
+  expect((await errorEvent)[0]!.message).toContain('not_caught')
```

Comment:
The error format between Bun and Node is different so can't directly match the error message.